.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "@frontside"
+  "extends": "@frontside/eslint-config"
```

Comment:
Is this out of band for this change?

Reply:
Oh, it's something that I just came across. How does eslint know which config to take just by extending `@frontside`?